{"error":{"code":101,"message":"Invalid username/password."},"level":"error","message":"Error generating response. ParseError { code: 101, message: 'Invalid username/password.' }","timestamp":"2016-10-24T01:28:42.584Z"}
{"error":{"code":101,"message":"Invalid username/password."},"level":"error","message":"Error generating response. ParseError { code: 101, message: 'Invalid username/password.' }","timestamp":"2016-10-24T01:39:28.443Z"}
{"error":{"name":"MongoError","message":"failed to connect to server [ds011389.mlab.com:11389] on first connect"},"level":"error","message":"Error generating response. { MongoError: failed to connect to server [ds011389.mlab.com:11389] on first connect\n    at Pool.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\topologies\\server.js:313:35)\n    at emitOne (events.js:96:13)\n    at Pool.emit (events.js:188:7)\n    at Connection.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\connection\\pool.js:260:12)\n    at Connection.g (events.js:291:16)\n    at emitTwo (events.js:106:13)\n    at Connection.emit (events.js:191:7)\n    at Socket.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\connection\\connection.js:162:49)\n    at Socket.g (events.js:291:16)\n    at emitOne (events.js:96:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [ds011389.mlab.com:11389] on first connect' }","timestamp":"2016-10-24T09:56:35.958Z"}
{"level":"error","message":"Uncaught internal server error. { MongoError: failed to connect to server [ds011389.mlab.com:11389] on first connect\n    at Pool.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\topologies\\server.js:313:35)\n    at emitOne (events.js:96:13)\n    at Pool.emit (events.js:188:7)\n    at Connection.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\connection\\pool.js:260:12)\n    at Connection.g (events.js:291:16)\n    at emitTwo (events.js:106:13)\n    at Connection.emit (events.js:191:7)\n    at Socket.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\connection\\connection.js:162:49)\n    at Socket.g (events.js:291:16)\n    at emitOne (events.js:96:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [ds011389.mlab.com:11389] on first connect' } MongoError: failed to connect to server [ds011389.mlab.com:11389] on first connect\n    at Pool.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\topologies\\server.js:313:35)\n    at emitOne (events.js:96:13)\n    at Pool.emit (events.js:188:7)\n    at Connection.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\connection\\pool.js:260:12)\n    at Connection.g (events.js:291:16)\n    at emitTwo (events.js:106:13)\n    at Connection.emit (events.js:191:7)\n    at Socket.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\connection\\connection.js:162:49)\n    at Socket.g (events.js:291:16)\n    at emitOne (events.js:96:13)","timestamp":"2016-10-24T09:56:35.961Z"}
{"error":{"name":"MongoError","message":"failed to connect to server [ds011389.mlab.com:11389] on first connect"},"level":"error","message":"Error generating response. { MongoError: failed to connect to server [ds011389.mlab.com:11389] on first connect\n    at Pool.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\topologies\\server.js:313:35)\n    at emitOne (events.js:96:13)\n    at Pool.emit (events.js:188:7)\n    at Connection.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\connection\\pool.js:260:12)\n    at Connection.g (events.js:291:16)\n    at emitTwo (events.js:106:13)\n    at Connection.emit (events.js:191:7)\n    at Socket.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\connection\\connection.js:162:49)\n    at Socket.g (events.js:291:16)\n    at emitOne (events.js:96:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [ds011389.mlab.com:11389] on first connect' }","timestamp":"2016-10-24T09:56:35.979Z"}
{"level":"error","message":"Uncaught internal server error. { MongoError: failed to connect to server [ds011389.mlab.com:11389] on first connect\n    at Pool.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\topologies\\server.js:313:35)\n    at emitOne (events.js:96:13)\n    at Pool.emit (events.js:188:7)\n    at Connection.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\connection\\pool.js:260:12)\n    at Connection.g (events.js:291:16)\n    at emitTwo (events.js:106:13)\n    at Connection.emit (events.js:191:7)\n    at Socket.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\connection\\connection.js:162:49)\n    at Socket.g (events.js:291:16)\n    at emitOne (events.js:96:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [ds011389.mlab.com:11389] on first connect' } MongoError: failed to connect to server [ds011389.mlab.com:11389] on first connect\n    at Pool.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\topologies\\server.js:313:35)\n    at emitOne (events.js:96:13)\n    at Pool.emit (events.js:188:7)\n    at Connection.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\connection\\pool.js:260:12)\n    at Connection.g (events.js:291:16)\n    at emitTwo (events.js:106:13)\n    at Connection.emit (events.js:191:7)\n    at Socket.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\connection\\connection.js:162:49)\n    at Socket.g (events.js:291:16)\n    at emitOne (events.js:96:13)","timestamp":"2016-10-24T09:56:35.979Z"}
{"error":{"name":"MongoError","message":"failed to connect to server [ds011389.mlab.com:11389] on first connect"},"level":"error","message":"Error generating response. { MongoError: failed to connect to server [ds011389.mlab.com:11389] on first connect\n    at Pool.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\topologies\\server.js:313:35)\n    at emitOne (events.js:96:13)\n    at Pool.emit (events.js:188:7)\n    at Connection.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\connection\\pool.js:260:12)\n    at Connection.g (events.js:291:16)\n    at emitTwo (events.js:106:13)\n    at Connection.emit (events.js:191:7)\n    at Socket.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\connection\\connection.js:162:49)\n    at Socket.g (events.js:291:16)\n    at emitOne (events.js:96:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [ds011389.mlab.com:11389] on first connect' }","timestamp":"2016-10-24T09:56:36.037Z"}
{"level":"error","message":"Uncaught internal server error. { MongoError: failed to connect to server [ds011389.mlab.com:11389] on first connect\n    at Pool.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\topologies\\server.js:313:35)\n    at emitOne (events.js:96:13)\n    at Pool.emit (events.js:188:7)\n    at Connection.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\connection\\pool.js:260:12)\n    at Connection.g (events.js:291:16)\n    at emitTwo (events.js:106:13)\n    at Connection.emit (events.js:191:7)\n    at Socket.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\connection\\connection.js:162:49)\n    at Socket.g (events.js:291:16)\n    at emitOne (events.js:96:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [ds011389.mlab.com:11389] on first connect' } MongoError: failed to connect to server [ds011389.mlab.com:11389] on first connect\n    at Pool.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\topologies\\server.js:313:35)\n    at emitOne (events.js:96:13)\n    at Pool.emit (events.js:188:7)\n    at Connection.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\connection\\pool.js:260:12)\n    at Connection.g (events.js:291:16)\n    at emitTwo (events.js:106:13)\n    at Connection.emit (events.js:191:7)\n    at Socket.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\connection\\connection.js:162:49)\n    at Socket.g (events.js:291:16)\n    at emitOne (events.js:96:13)","timestamp":"2016-10-24T09:56:36.037Z"}
{"error":{"name":"MongoError","message":"failed to connect to server [ds011389.mlab.com:11389] on first connect"},"level":"error","message":"Error generating response. { MongoError: failed to connect to server [ds011389.mlab.com:11389] on first connect\n    at Pool.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\topologies\\server.js:313:35)\n    at emitOne (events.js:96:13)\n    at Pool.emit (events.js:188:7)\n    at Connection.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\connection\\pool.js:260:12)\n    at Connection.g (events.js:291:16)\n    at emitTwo (events.js:106:13)\n    at Connection.emit (events.js:191:7)\n    at Socket.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\connection\\connection.js:162:49)\n    at Socket.g (events.js:291:16)\n    at emitOne (events.js:96:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [ds011389.mlab.com:11389] on first connect' }","timestamp":"2016-10-24T09:56:36.076Z"}
{"level":"error","message":"Uncaught internal server error. { MongoError: failed to connect to server [ds011389.mlab.com:11389] on first connect\n    at Pool.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\topologies\\server.js:313:35)\n    at emitOne (events.js:96:13)\n    at Pool.emit (events.js:188:7)\n    at Connection.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\connection\\pool.js:260:12)\n    at Connection.g (events.js:291:16)\n    at emitTwo (events.js:106:13)\n    at Connection.emit (events.js:191:7)\n    at Socket.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\connection\\connection.js:162:49)\n    at Socket.g (events.js:291:16)\n    at emitOne (events.js:96:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [ds011389.mlab.com:11389] on first connect' } MongoError: failed to connect to server [ds011389.mlab.com:11389] on first connect\n    at Pool.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\topologies\\server.js:313:35)\n    at emitOne (events.js:96:13)\n    at Pool.emit (events.js:188:7)\n    at Connection.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\connection\\pool.js:260:12)\n    at Connection.g (events.js:291:16)\n    at emitTwo (events.js:106:13)\n    at Connection.emit (events.js:191:7)\n    at Socket.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\connection\\connection.js:162:49)\n    at Socket.g (events.js:291:16)\n    at emitOne (events.js:96:13)","timestamp":"2016-10-24T09:56:36.077Z"}
{"error":{"name":"MongoError","message":"failed to connect to server [ds011389.mlab.com:11389] on first connect"},"level":"error","message":"Error generating response. { MongoError: failed to connect to server [ds011389.mlab.com:11389] on first connect\n    at Pool.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\topologies\\server.js:313:35)\n    at emitOne (events.js:96:13)\n    at Pool.emit (events.js:188:7)\n    at Connection.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\connection\\pool.js:260:12)\n    at Connection.g (events.js:291:16)\n    at emitTwo (events.js:106:13)\n    at Connection.emit (events.js:191:7)\n    at Socket.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\connection\\connection.js:162:49)\n    at Socket.g (events.js:291:16)\n    at emitOne (events.js:96:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [ds011389.mlab.com:11389] on first connect' }","timestamp":"2016-10-24T09:56:36.373Z"}
{"level":"error","message":"Uncaught internal server error. { MongoError: failed to connect to server [ds011389.mlab.com:11389] on first connect\n    at Pool.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\topologies\\server.js:313:35)\n    at emitOne (events.js:96:13)\n    at Pool.emit (events.js:188:7)\n    at Connection.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\connection\\pool.js:260:12)\n    at Connection.g (events.js:291:16)\n    at emitTwo (events.js:106:13)\n    at Connection.emit (events.js:191:7)\n    at Socket.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\connection\\connection.js:162:49)\n    at Socket.g (events.js:291:16)\n    at emitOne (events.js:96:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [ds011389.mlab.com:11389] on first connect' } MongoError: failed to connect to server [ds011389.mlab.com:11389] on first connect\n    at Pool.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\topologies\\server.js:313:35)\n    at emitOne (events.js:96:13)\n    at Pool.emit (events.js:188:7)\n    at Connection.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\connection\\pool.js:260:12)\n    at Connection.g (events.js:291:16)\n    at emitTwo (events.js:106:13)\n    at Connection.emit (events.js:191:7)\n    at Socket.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\connection\\connection.js:162:49)\n    at Socket.g (events.js:291:16)\n    at emitOne (events.js:96:13)","timestamp":"2016-10-24T09:56:36.373Z"}
{"error":{"name":"MongoError","message":"failed to connect to server [ds011389.mlab.com:11389] on first connect"},"level":"error","message":"Error generating response. { MongoError: failed to connect to server [ds011389.mlab.com:11389] on first connect\n    at Pool.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\topologies\\server.js:313:35)\n    at emitOne (events.js:96:13)\n    at Pool.emit (events.js:188:7)\n    at Connection.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\connection\\pool.js:260:12)\n    at Connection.g (events.js:291:16)\n    at emitTwo (events.js:106:13)\n    at Connection.emit (events.js:191:7)\n    at Socket.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\connection\\connection.js:162:49)\n    at Socket.g (events.js:291:16)\n    at emitOne (events.js:96:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [ds011389.mlab.com:11389] on first connect' }","timestamp":"2016-10-24T09:56:36.524Z"}
{"level":"error","message":"Uncaught internal server error. { MongoError: failed to connect to server [ds011389.mlab.com:11389] on first connect\n    at Pool.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\topologies\\server.js:313:35)\n    at emitOne (events.js:96:13)\n    at Pool.emit (events.js:188:7)\n    at Connection.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\connection\\pool.js:260:12)\n    at Connection.g (events.js:291:16)\n    at emitTwo (events.js:106:13)\n    at Connection.emit (events.js:191:7)\n    at Socket.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\connection\\connection.js:162:49)\n    at Socket.g (events.js:291:16)\n    at emitOne (events.js:96:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [ds011389.mlab.com:11389] on first connect' } MongoError: failed to connect to server [ds011389.mlab.com:11389] on first connect\n    at Pool.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\topologies\\server.js:313:35)\n    at emitOne (events.js:96:13)\n    at Pool.emit (events.js:188:7)\n    at Connection.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\connection\\pool.js:260:12)\n    at Connection.g (events.js:291:16)\n    at emitTwo (events.js:106:13)\n    at Connection.emit (events.js:191:7)\n    at Socket.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\connection\\connection.js:162:49)\n    at Socket.g (events.js:291:16)\n    at emitOne (events.js:96:13)","timestamp":"2016-10-24T09:56:36.524Z"}
{"error":{"name":"MongoError","message":"failed to connect to server [ds011389.mlab.com:11389] on first connect"},"level":"error","message":"Error generating response. { MongoError: failed to connect to server [ds011389.mlab.com:11389] on first connect\n    at Pool.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\topologies\\server.js:313:35)\n    at emitOne (events.js:96:13)\n    at Pool.emit (events.js:188:7)\n    at Connection.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\connection\\pool.js:260:12)\n    at Connection.g (events.js:291:16)\n    at emitTwo (events.js:106:13)\n    at Connection.emit (events.js:191:7)\n    at Socket.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\connection\\connection.js:162:49)\n    at Socket.g (events.js:291:16)\n    at emitOne (events.js:96:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [ds011389.mlab.com:11389] on first connect' }","timestamp":"2016-10-24T09:56:36.541Z"}
{"level":"error","message":"Uncaught internal server error. { MongoError: failed to connect to server [ds011389.mlab.com:11389] on first connect\n    at Pool.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\topologies\\server.js:313:35)\n    at emitOne (events.js:96:13)\n    at Pool.emit (events.js:188:7)\n    at Connection.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\connection\\pool.js:260:12)\n    at Connection.g (events.js:291:16)\n    at emitTwo (events.js:106:13)\n    at Connection.emit (events.js:191:7)\n    at Socket.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\connection\\connection.js:162:49)\n    at Socket.g (events.js:291:16)\n    at emitOne (events.js:96:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [ds011389.mlab.com:11389] on first connect' } MongoError: failed to connect to server [ds011389.mlab.com:11389] on first connect\n    at Pool.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\topologies\\server.js:313:35)\n    at emitOne (events.js:96:13)\n    at Pool.emit (events.js:188:7)\n    at Connection.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\connection\\pool.js:260:12)\n    at Connection.g (events.js:291:16)\n    at emitTwo (events.js:106:13)\n    at Connection.emit (events.js:191:7)\n    at Socket.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\connection\\connection.js:162:49)\n    at Socket.g (events.js:291:16)\n    at emitOne (events.js:96:13)","timestamp":"2016-10-24T09:56:36.542Z"}
{"error":{"name":"MongoError","message":"failed to connect to server [ds011389.mlab.com:11389] on first connect"},"level":"error","message":"Error generating response. { MongoError: failed to connect to server [ds011389.mlab.com:11389] on first connect\n    at Pool.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\topologies\\server.js:313:35)\n    at emitOne (events.js:96:13)\n    at Pool.emit (events.js:188:7)\n    at Connection.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\connection\\pool.js:260:12)\n    at Connection.g (events.js:291:16)\n    at emitTwo (events.js:106:13)\n    at Connection.emit (events.js:191:7)\n    at Socket.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\connection\\connection.js:162:49)\n    at Socket.g (events.js:291:16)\n    at emitOne (events.js:96:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [ds011389.mlab.com:11389] on first connect' }","timestamp":"2016-10-24T09:56:37.381Z"}
{"level":"error","message":"Uncaught internal server error. { MongoError: failed to connect to server [ds011389.mlab.com:11389] on first connect\n    at Pool.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\topologies\\server.js:313:35)\n    at emitOne (events.js:96:13)\n    at Pool.emit (events.js:188:7)\n    at Connection.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\connection\\pool.js:260:12)\n    at Connection.g (events.js:291:16)\n    at emitTwo (events.js:106:13)\n    at Connection.emit (events.js:191:7)\n    at Socket.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\connection\\connection.js:162:49)\n    at Socket.g (events.js:291:16)\n    at emitOne (events.js:96:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [ds011389.mlab.com:11389] on first connect' } MongoError: failed to connect to server [ds011389.mlab.com:11389] on first connect\n    at Pool.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\topologies\\server.js:313:35)\n    at emitOne (events.js:96:13)\n    at Pool.emit (events.js:188:7)\n    at Connection.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\connection\\pool.js:260:12)\n    at Connection.g (events.js:291:16)\n    at emitTwo (events.js:106:13)\n    at Connection.emit (events.js:191:7)\n    at Socket.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\connection\\connection.js:162:49)\n    at Socket.g (events.js:291:16)\n    at emitOne (events.js:96:13)","timestamp":"2016-10-24T09:56:37.381Z"}
{"error":{"name":"MongoError","message":"failed to connect to server [ds011389.mlab.com:11389] on first connect"},"level":"error","message":"Error generating response. { MongoError: failed to connect to server [ds011389.mlab.com:11389] on first connect\n    at Pool.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\topologies\\server.js:313:35)\n    at emitOne (events.js:96:13)\n    at Pool.emit (events.js:188:7)\n    at Connection.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\connection\\pool.js:260:12)\n    at Connection.g (events.js:291:16)\n    at emitTwo (events.js:106:13)\n    at Connection.emit (events.js:191:7)\n    at Socket.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\connection\\connection.js:162:49)\n    at Socket.g (events.js:291:16)\n    at emitOne (events.js:96:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [ds011389.mlab.com:11389] on first connect' }","timestamp":"2016-10-24T09:56:38.221Z"}
{"level":"error","message":"Uncaught internal server error. { MongoError: failed to connect to server [ds011389.mlab.com:11389] on first connect\n    at Pool.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\topologies\\server.js:313:35)\n    at emitOne (events.js:96:13)\n    at Pool.emit (events.js:188:7)\n    at Connection.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\connection\\pool.js:260:12)\n    at Connection.g (events.js:291:16)\n    at emitTwo (events.js:106:13)\n    at Connection.emit (events.js:191:7)\n    at Socket.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\connection\\connection.js:162:49)\n    at Socket.g (events.js:291:16)\n    at emitOne (events.js:96:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [ds011389.mlab.com:11389] on first connect' } MongoError: failed to connect to server [ds011389.mlab.com:11389] on first connect\n    at Pool.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\topologies\\server.js:313:35)\n    at emitOne (events.js:96:13)\n    at Pool.emit (events.js:188:7)\n    at Connection.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\connection\\pool.js:260:12)\n    at Connection.g (events.js:291:16)\n    at emitTwo (events.js:106:13)\n    at Connection.emit (events.js:191:7)\n    at Socket.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\connection\\connection.js:162:49)\n    at Socket.g (events.js:291:16)\n    at emitOne (events.js:96:13)","timestamp":"2016-10-24T09:56:38.221Z"}
{"error":{"name":"MongoError","message":"failed to connect to server [ds011389.mlab.com:11389] on first connect"},"level":"error","message":"Error generating response. { MongoError: failed to connect to server [ds011389.mlab.com:11389] on first connect\n    at Pool.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\topologies\\server.js:313:35)\n    at emitOne (events.js:96:13)\n    at Pool.emit (events.js:188:7)\n    at Connection.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\connection\\pool.js:260:12)\n    at Connection.g (events.js:291:16)\n    at emitTwo (events.js:106:13)\n    at Connection.emit (events.js:191:7)\n    at Socket.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\connection\\connection.js:162:49)\n    at Socket.g (events.js:291:16)\n    at emitOne (events.js:96:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [ds011389.mlab.com:11389] on first connect' }","timestamp":"2016-10-24T09:56:38.699Z"}
{"level":"error","message":"Uncaught internal server error. { MongoError: failed to connect to server [ds011389.mlab.com:11389] on first connect\n    at Pool.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\topologies\\server.js:313:35)\n    at emitOne (events.js:96:13)\n    at Pool.emit (events.js:188:7)\n    at Connection.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\connection\\pool.js:260:12)\n    at Connection.g (events.js:291:16)\n    at emitTwo (events.js:106:13)\n    at Connection.emit (events.js:191:7)\n    at Socket.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\connection\\connection.js:162:49)\n    at Socket.g (events.js:291:16)\n    at emitOne (events.js:96:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [ds011389.mlab.com:11389] on first connect' } MongoError: failed to connect to server [ds011389.mlab.com:11389] on first connect\n    at Pool.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\topologies\\server.js:313:35)\n    at emitOne (events.js:96:13)\n    at Pool.emit (events.js:188:7)\n    at Connection.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\connection\\pool.js:260:12)\n    at Connection.g (events.js:291:16)\n    at emitTwo (events.js:106:13)\n    at Connection.emit (events.js:191:7)\n    at Socket.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\connection\\connection.js:162:49)\n    at Socket.g (events.js:291:16)\n    at emitOne (events.js:96:13)","timestamp":"2016-10-24T09:56:38.699Z"}
{"error":{"name":"MongoError","message":"failed to connect to server [ds011389.mlab.com:11389] on first connect"},"level":"error","message":"Error generating response. { MongoError: failed to connect to server [ds011389.mlab.com:11389] on first connect\n    at Pool.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\topologies\\server.js:313:35)\n    at emitOne (events.js:96:13)\n    at Pool.emit (events.js:188:7)\n    at Connection.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\connection\\pool.js:260:12)\n    at Connection.g (events.js:291:16)\n    at emitTwo (events.js:106:13)\n    at Connection.emit (events.js:191:7)\n    at Socket.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\connection\\connection.js:162:49)\n    at Socket.g (events.js:291:16)\n    at emitOne (events.js:96:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [ds011389.mlab.com:11389] on first connect' }","timestamp":"2016-10-24T09:56:39.893Z"}
{"level":"error","message":"Uncaught internal server error. { MongoError: failed to connect to server [ds011389.mlab.com:11389] on first connect\n    at Pool.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\topologies\\server.js:313:35)\n    at emitOne (events.js:96:13)\n    at Pool.emit (events.js:188:7)\n    at Connection.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\connection\\pool.js:260:12)\n    at Connection.g (events.js:291:16)\n    at emitTwo (events.js:106:13)\n    at Connection.emit (events.js:191:7)\n    at Socket.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\connection\\connection.js:162:49)\n    at Socket.g (events.js:291:16)\n    at emitOne (events.js:96:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [ds011389.mlab.com:11389] on first connect' } MongoError: failed to connect to server [ds011389.mlab.com:11389] on first connect\n    at Pool.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\topologies\\server.js:313:35)\n    at emitOne (events.js:96:13)\n    at Pool.emit (events.js:188:7)\n    at Connection.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\connection\\pool.js:260:12)\n    at Connection.g (events.js:291:16)\n    at emitTwo (events.js:106:13)\n    at Connection.emit (events.js:191:7)\n    at Socket.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\connection\\connection.js:162:49)\n    at Socket.g (events.js:291:16)\n    at emitOne (events.js:96:13)","timestamp":"2016-10-24T09:56:39.893Z"}
{"error":{"name":"MongoError","message":"failed to connect to server [ds011389.mlab.com:11389] on first connect"},"level":"error","message":"Error generating response. { MongoError: failed to connect to server [ds011389.mlab.com:11389] on first connect\n    at Pool.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\topologies\\server.js:313:35)\n    at emitOne (events.js:96:13)\n    at Pool.emit (events.js:188:7)\n    at Connection.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\connection\\pool.js:260:12)\n    at Connection.g (events.js:291:16)\n    at emitTwo (events.js:106:13)\n    at Connection.emit (events.js:191:7)\n    at Socket.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\connection\\connection.js:162:49)\n    at Socket.g (events.js:291:16)\n    at emitOne (events.js:96:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [ds011389.mlab.com:11389] on first connect' }","timestamp":"2016-10-24T09:56:39.919Z"}
{"level":"error","message":"Uncaught internal server error. { MongoError: failed to connect to server [ds011389.mlab.com:11389] on first connect\n    at Pool.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\topologies\\server.js:313:35)\n    at emitOne (events.js:96:13)\n    at Pool.emit (events.js:188:7)\n    at Connection.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\connection\\pool.js:260:12)\n    at Connection.g (events.js:291:16)\n    at emitTwo (events.js:106:13)\n    at Connection.emit (events.js:191:7)\n    at Socket.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\connection\\connection.js:162:49)\n    at Socket.g (events.js:291:16)\n    at emitOne (events.js:96:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [ds011389.mlab.com:11389] on first connect' } MongoError: failed to connect to server [ds011389.mlab.com:11389] on first connect\n    at Pool.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\topologies\\server.js:313:35)\n    at emitOne (events.js:96:13)\n    at Pool.emit (events.js:188:7)\n    at Connection.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\connection\\pool.js:260:12)\n    at Connection.g (events.js:291:16)\n    at emitTwo (events.js:106:13)\n    at Connection.emit (events.js:191:7)\n    at Socket.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\connection\\connection.js:162:49)\n    at Socket.g (events.js:291:16)\n    at emitOne (events.js:96:13)","timestamp":"2016-10-24T09:56:39.920Z"}
{"error":{"name":"MongoError","message":"failed to connect to server [ds011389.mlab.com:11389] on first connect"},"level":"error","message":"Error generating response. { MongoError: failed to connect to server [ds011389.mlab.com:11389] on first connect\n    at Pool.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\topologies\\server.js:313:35)\n    at emitOne (events.js:96:13)\n    at Pool.emit (events.js:188:7)\n    at Connection.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\connection\\pool.js:260:12)\n    at Connection.g (events.js:291:16)\n    at emitTwo (events.js:106:13)\n    at Connection.emit (events.js:191:7)\n    at Socket.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\connection\\connection.js:162:49)\n    at Socket.g (events.js:291:16)\n    at emitOne (events.js:96:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [ds011389.mlab.com:11389] on first connect' }","timestamp":"2016-10-24T09:56:40.082Z"}
{"level":"error","message":"Uncaught internal server error. { MongoError: failed to connect to server [ds011389.mlab.com:11389] on first connect\n    at Pool.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\topologies\\server.js:313:35)\n    at emitOne (events.js:96:13)\n    at Pool.emit (events.js:188:7)\n    at Connection.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\connection\\pool.js:260:12)\n    at Connection.g (events.js:291:16)\n    at emitTwo (events.js:106:13)\n    at Connection.emit (events.js:191:7)\n    at Socket.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\connection\\connection.js:162:49)\n    at Socket.g (events.js:291:16)\n    at emitOne (events.js:96:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [ds011389.mlab.com:11389] on first connect' } MongoError: failed to connect to server [ds011389.mlab.com:11389] on first connect\n    at Pool.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\topologies\\server.js:313:35)\n    at emitOne (events.js:96:13)\n    at Pool.emit (events.js:188:7)\n    at Connection.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\connection\\pool.js:260:12)\n    at Connection.g (events.js:291:16)\n    at emitTwo (events.js:106:13)\n    at Connection.emit (events.js:191:7)\n    at Socket.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\connection\\connection.js:162:49)\n    at Socket.g (events.js:291:16)\n    at emitOne (events.js:96:13)","timestamp":"2016-10-24T09:56:40.082Z"}
{"error":{"name":"MongoError","message":"failed to connect to server [ds011389.mlab.com:11389] on first connect"},"level":"error","message":"Error generating response. { MongoError: failed to connect to server [ds011389.mlab.com:11389] on first connect\n    at Pool.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\topologies\\server.js:313:35)\n    at emitOne (events.js:96:13)\n    at Pool.emit (events.js:188:7)\n    at Connection.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\connection\\pool.js:260:12)\n    at Connection.g (events.js:291:16)\n    at emitTwo (events.js:106:13)\n    at Connection.emit (events.js:191:7)\n    at Socket.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\connection\\connection.js:162:49)\n    at Socket.g (events.js:291:16)\n    at emitOne (events.js:96:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [ds011389.mlab.com:11389] on first connect' }","timestamp":"2016-10-24T09:56:40.092Z"}
{"level":"error","message":"Uncaught internal server error. { MongoError: failed to connect to server [ds011389.mlab.com:11389] on first connect\n    at Pool.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\topologies\\server.js:313:35)\n    at emitOne (events.js:96:13)\n    at Pool.emit (events.js:188:7)\n    at Connection.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\connection\\pool.js:260:12)\n    at Connection.g (events.js:291:16)\n    at emitTwo (events.js:106:13)\n    at Connection.emit (events.js:191:7)\n    at Socket.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\connection\\connection.js:162:49)\n    at Socket.g (events.js:291:16)\n    at emitOne (events.js:96:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [ds011389.mlab.com:11389] on first connect' } MongoError: failed to connect to server [ds011389.mlab.com:11389] on first connect\n    at Pool.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\topologies\\server.js:313:35)\n    at emitOne (events.js:96:13)\n    at Pool.emit (events.js:188:7)\n    at Connection.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\connection\\pool.js:260:12)\n    at Connection.g (events.js:291:16)\n    at emitTwo (events.js:106:13)\n    at Connection.emit (events.js:191:7)\n    at Socket.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\connection\\connection.js:162:49)\n    at Socket.g (events.js:291:16)\n    at emitOne (events.js:96:13)","timestamp":"2016-10-24T09:56:40.093Z"}
{"error":{"name":"MongoError","message":"failed to connect to server [ds011389.mlab.com:11389] on first connect"},"level":"error","message":"Error generating response. { MongoError: failed to connect to server [ds011389.mlab.com:11389] on first connect\n    at Pool.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\topologies\\server.js:313:35)\n    at emitOne (events.js:96:13)\n    at Pool.emit (events.js:188:7)\n    at Connection.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\connection\\pool.js:260:12)\n    at Connection.g (events.js:291:16)\n    at emitTwo (events.js:106:13)\n    at Connection.emit (events.js:191:7)\n    at Socket.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\connection\\connection.js:162:49)\n    at Socket.g (events.js:291:16)\n    at emitOne (events.js:96:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [ds011389.mlab.com:11389] on first connect' }","timestamp":"2016-10-24T09:56:40.108Z"}
{"level":"error","message":"Uncaught internal server error. { MongoError: failed to connect to server [ds011389.mlab.com:11389] on first connect\n    at Pool.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\topologies\\server.js:313:35)\n    at emitOne (events.js:96:13)\n    at Pool.emit (events.js:188:7)\n    at Connection.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\connection\\pool.js:260:12)\n    at Connection.g (events.js:291:16)\n    at emitTwo (events.js:106:13)\n    at Connection.emit (events.js:191:7)\n    at Socket.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\connection\\connection.js:162:49)\n    at Socket.g (events.js:291:16)\n    at emitOne (events.js:96:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [ds011389.mlab.com:11389] on first connect' } MongoError: failed to connect to server [ds011389.mlab.com:11389] on first connect\n    at Pool.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\topologies\\server.js:313:35)\n    at emitOne (events.js:96:13)\n    at Pool.emit (events.js:188:7)\n    at Connection.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\connection\\pool.js:260:12)\n    at Connection.g (events.js:291:16)\n    at emitTwo (events.js:106:13)\n    at Connection.emit (events.js:191:7)\n    at Socket.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\connection\\connection.js:162:49)\n    at Socket.g (events.js:291:16)\n    at emitOne (events.js:96:13)","timestamp":"2016-10-24T09:56:40.108Z"}
{"error":{"name":"MongoError","message":"failed to connect to server [ds011389.mlab.com:11389] on first connect"},"level":"error","message":"Error generating response. { MongoError: failed to connect to server [ds011389.mlab.com:11389] on first connect\n    at Pool.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\topologies\\server.js:313:35)\n    at emitOne (events.js:96:13)\n    at Pool.emit (events.js:188:7)\n    at Connection.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\connection\\pool.js:260:12)\n    at Connection.g (events.js:291:16)\n    at emitTwo (events.js:106:13)\n    at Connection.emit (events.js:191:7)\n    at Socket.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\connection\\connection.js:162:49)\n    at Socket.g (events.js:291:16)\n    at emitOne (events.js:96:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [ds011389.mlab.com:11389] on first connect' }","timestamp":"2016-10-24T09:56:40.179Z"}
{"level":"error","message":"Uncaught internal server error. { MongoError: failed to connect to server [ds011389.mlab.com:11389] on first connect\n    at Pool.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\topologies\\server.js:313:35)\n    at emitOne (events.js:96:13)\n    at Pool.emit (events.js:188:7)\n    at Connection.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\connection\\pool.js:260:12)\n    at Connection.g (events.js:291:16)\n    at emitTwo (events.js:106:13)\n    at Connection.emit (events.js:191:7)\n    at Socket.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\connection\\connection.js:162:49)\n    at Socket.g (events.js:291:16)\n    at emitOne (events.js:96:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [ds011389.mlab.com:11389] on first connect' } MongoError: failed to connect to server [ds011389.mlab.com:11389] on first connect\n    at Pool.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\topologies\\server.js:313:35)\n    at emitOne (events.js:96:13)\n    at Pool.emit (events.js:188:7)\n    at Connection.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\connection\\pool.js:260:12)\n    at Connection.g (events.js:291:16)\n    at emitTwo (events.js:106:13)\n    at Connection.emit (events.js:191:7)\n    at Socket.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\connection\\connection.js:162:49)\n    at Socket.g (events.js:291:16)\n    at emitOne (events.js:96:13)","timestamp":"2016-10-24T09:56:40.179Z"}
{"error":{"name":"MongoError","message":"failed to connect to server [ds011389.mlab.com:11389] on first connect"},"level":"error","message":"Error generating response. { MongoError: failed to connect to server [ds011389.mlab.com:11389] on first connect\n    at Pool.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\topologies\\server.js:313:35)\n    at emitOne (events.js:96:13)\n    at Pool.emit (events.js:188:7)\n    at Connection.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\connection\\pool.js:260:12)\n    at Connection.g (events.js:291:16)\n    at emitTwo (events.js:106:13)\n    at Connection.emit (events.js:191:7)\n    at Socket.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\connection\\connection.js:162:49)\n    at Socket.g (events.js:291:16)\n    at emitOne (events.js:96:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [ds011389.mlab.com:11389] on first connect' }","timestamp":"2016-10-24T09:56:40.275Z"}
{"level":"error","message":"Uncaught internal server error. { MongoError: failed to connect to server [ds011389.mlab.com:11389] on first connect\n    at Pool.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\topologies\\server.js:313:35)\n    at emitOne (events.js:96:13)\n    at Pool.emit (events.js:188:7)\n    at Connection.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\connection\\pool.js:260:12)\n    at Connection.g (events.js:291:16)\n    at emitTwo (events.js:106:13)\n    at Connection.emit (events.js:191:7)\n    at Socket.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\connection\\connection.js:162:49)\n    at Socket.g (events.js:291:16)\n    at emitOne (events.js:96:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [ds011389.mlab.com:11389] on first connect' } MongoError: failed to connect to server [ds011389.mlab.com:11389] on first connect\n    at Pool.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\topologies\\server.js:313:35)\n    at emitOne (events.js:96:13)\n    at Pool.emit (events.js:188:7)\n    at Connection.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\connection\\pool.js:260:12)\n    at Connection.g (events.js:291:16)\n    at emitTwo (events.js:106:13)\n    at Connection.emit (events.js:191:7)\n    at Socket.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\connection\\connection.js:162:49)\n    at Socket.g (events.js:291:16)\n    at emitOne (events.js:96:13)","timestamp":"2016-10-24T09:56:40.275Z"}
{"error":{"name":"MongoError","message":"failed to connect to server [ds011389.mlab.com:11389] on first connect"},"level":"error","message":"Error generating response. { MongoError: failed to connect to server [ds011389.mlab.com:11389] on first connect\n    at Pool.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\topologies\\server.js:313:35)\n    at emitOne (events.js:96:13)\n    at Pool.emit (events.js:188:7)\n    at Connection.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\connection\\pool.js:260:12)\n    at Connection.g (events.js:291:16)\n    at emitTwo (events.js:106:13)\n    at Connection.emit (events.js:191:7)\n    at Socket.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\connection\\connection.js:162:49)\n    at Socket.g (events.js:291:16)\n    at emitOne (events.js:96:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [ds011389.mlab.com:11389] on first connect' }","timestamp":"2016-10-24T09:56:40.634Z"}
{"level":"error","message":"Uncaught internal server error. { MongoError: failed to connect to server [ds011389.mlab.com:11389] on first connect\n    at Pool.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\topologies\\server.js:313:35)\n    at emitOne (events.js:96:13)\n    at Pool.emit (events.js:188:7)\n    at Connection.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\connection\\pool.js:260:12)\n    at Connection.g (events.js:291:16)\n    at emitTwo (events.js:106:13)\n    at Connection.emit (events.js:191:7)\n    at Socket.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\connection\\connection.js:162:49)\n    at Socket.g (events.js:291:16)\n    at emitOne (events.js:96:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [ds011389.mlab.com:11389] on first connect' } MongoError: failed to connect to server [ds011389.mlab.com:11389] on first connect\n    at Pool.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\topologies\\server.js:313:35)\n    at emitOne (events.js:96:13)\n    at Pool.emit (events.js:188:7)\n    at Connection.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\connection\\pool.js:260:12)\n    at Connection.g (events.js:291:16)\n    at emitTwo (events.js:106:13)\n    at Connection.emit (events.js:191:7)\n    at Socket.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\connection\\connection.js:162:49)\n    at Socket.g (events.js:291:16)\n    at emitOne (events.js:96:13)","timestamp":"2016-10-24T09:56:40.634Z"}
{"error":{"name":"MongoError","message":"failed to connect to server [ds011389.mlab.com:11389] on first connect"},"level":"error","message":"Error generating response. { MongoError: failed to connect to server [ds011389.mlab.com:11389] on first connect\n    at Pool.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\topologies\\server.js:313:35)\n    at emitOne (events.js:96:13)\n    at Pool.emit (events.js:188:7)\n    at Connection.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\connection\\pool.js:260:12)\n    at Connection.g (events.js:291:16)\n    at emitTwo (events.js:106:13)\n    at Connection.emit (events.js:191:7)\n    at Socket.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\connection\\connection.js:162:49)\n    at Socket.g (events.js:291:16)\n    at emitOne (events.js:96:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [ds011389.mlab.com:11389] on first connect' }","timestamp":"2016-10-24T09:56:41.794Z"}
{"level":"error","message":"Uncaught internal server error. { MongoError: failed to connect to server [ds011389.mlab.com:11389] on first connect\n    at Pool.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\topologies\\server.js:313:35)\n    at emitOne (events.js:96:13)\n    at Pool.emit (events.js:188:7)\n    at Connection.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\connection\\pool.js:260:12)\n    at Connection.g (events.js:291:16)\n    at emitTwo (events.js:106:13)\n    at Connection.emit (events.js:191:7)\n    at Socket.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\connection\\connection.js:162:49)\n    at Socket.g (events.js:291:16)\n    at emitOne (events.js:96:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [ds011389.mlab.com:11389] on first connect' } MongoError: failed to connect to server [ds011389.mlab.com:11389] on first connect\n    at Pool.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\topologies\\server.js:313:35)\n    at emitOne (events.js:96:13)\n    at Pool.emit (events.js:188:7)\n    at Connection.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\connection\\pool.js:260:12)\n    at Connection.g (events.js:291:16)\n    at emitTwo (events.js:106:13)\n    at Connection.emit (events.js:191:7)\n    at Socket.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\connection\\connection.js:162:49)\n    at Socket.g (events.js:291:16)\n    at emitOne (events.js:96:13)","timestamp":"2016-10-24T09:56:41.794Z"}
{"error":{"name":"MongoError","message":"failed to connect to server [ds011389.mlab.com:11389] on first connect"},"level":"error","message":"Error generating response. { MongoError: failed to connect to server [ds011389.mlab.com:11389] on first connect\n    at Pool.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\topologies\\server.js:313:35)\n    at emitOne (events.js:96:13)\n    at Pool.emit (events.js:188:7)\n    at Connection.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\connection\\pool.js:260:12)\n    at Connection.g (events.js:291:16)\n    at emitTwo (events.js:106:13)\n    at Connection.emit (events.js:191:7)\n    at Socket.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\connection\\connection.js:162:49)\n    at Socket.g (events.js:291:16)\n    at emitOne (events.js:96:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [ds011389.mlab.com:11389] on first connect' }","timestamp":"2016-10-24T09:56:43.762Z"}
{"level":"error","message":"Uncaught internal server error. { MongoError: failed to connect to server [ds011389.mlab.com:11389] on first connect\n    at Pool.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\topologies\\server.js:313:35)\n    at emitOne (events.js:96:13)\n    at Pool.emit (events.js:188:7)\n    at Connection.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\connection\\pool.js:260:12)\n    at Connection.g (events.js:291:16)\n    at emitTwo (events.js:106:13)\n    at Connection.emit (events.js:191:7)\n    at Socket.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\connection\\connection.js:162:49)\n    at Socket.g (events.js:291:16)\n    at emitOne (events.js:96:13)\n  name: 'MongoError',\n  message: 'failed to connect to server [ds011389.mlab.com:11389] on first connect' } MongoError: failed to connect to server [ds011389.mlab.com:11389] on first connect\n    at Pool.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\topologies\\server.js:313:35)\n    at emitOne (events.js:96:13)\n    at Pool.emit (events.js:188:7)\n    at Connection.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\connection\\pool.js:260:12)\n    at Connection.g (events.js:291:16)\n    at emitTwo (events.js:106:13)\n    at Connection.emit (events.js:191:7)\n    at Socket.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\connection\\connection.js:162:49)\n    at Socket.g (events.js:291:16)\n    at emitOne (events.js:96:13)","timestamp":"2016-10-24T09:56:43.763Z"}
{"error":{"code":202,"message":"Account already exists for this username."},"level":"error","message":"Error generating response. ParseError {\n  code: 202,\n  message: 'Account already exists for this username.' }","timestamp":"2016-10-24T16:47:48.699Z"}
{"error":{"code":202,"message":"Account already exists for this username."},"level":"error","message":"Error generating response. ParseError {\n  code: 202,\n  message: 'Account already exists for this username.' }","timestamp":"2016-10-24T16:49:16.238Z"}
{"message":"error getting auth for sessionToken","name":"MongoError","stack":"MongoError: connection 46 to ds011389.mlab.com:11389 timed out\n    at Function.MongoError.create (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\error.js:29:11)\n    at Socket.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\connection\\connection.js:173:20)\n    at Socket.g (events.js:291:16)\n    at emitNone (events.js:86:13)\n    at Socket.emit (events.js:185:7)\n    at Socket._onTimeout (net.js:334:8)\n    at tryOnTimeout (timers.js:232:11)\n    at Timer.listOnTimeout (timers.js:202:5)","level":"error","timestamp":"2016-10-24T18:33:52.480Z"}
{"message":"error getting auth for sessionToken","name":"MongoError","stack":"MongoError: connection 88 to ds011389.mlab.com:11389 timed out\n    at Function.MongoError.create (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\error.js:29:11)\n    at Socket.<anonymous> (C:\\Users\\ahmedengu\\IdeaProjects\\GaraParse\\node_modules\\mongodb-core\\lib\\connection\\connection.js:173:20)\n    at Socket.g (events.js:291:16)\n    at emitNone (events.js:86:13)\n    at Socket.emit (events.js:185:7)\n    at Socket._onTimeout (net.js:334:8)\n    at tryOnTimeout (timers.js:232:11)\n    at Timer.listOnTimeout (timers.js:202:5)","level":"error","timestamp":"2016-10-24T19:00:11.867Z"}
{"error":{"code":101,"message":"Object not found."},"level":"error","message":"Error generating response. ParseError { code: 101, message: 'Object not found.' }","timestamp":"2016-10-24T21:37:05.243Z"}
{"error":{"code":101,"message":"Object not found."},"level":"error","message":"Error generating response. ParseError { code: 101, message: 'Object not found.' }","timestamp":"2016-10-24T21:37:05.439Z"}
{"error":{"code":101,"message":"Object not found."},"level":"error","message":"Error generating response. ParseError { code: 101, message: 'Object not found.' }","timestamp":"2016-10-24T21:37:05.986Z"}
{"error":{"code":101,"message":"Object not found."},"level":"error","message":"Error generating response. ParseError { code: 101, message: 'Object not found.' }","timestamp":"2016-10-24T21:37:06.240Z"}
{"error":{"code":101,"message":"Object not found."},"level":"error","message":"Error generating response. ParseError { code: 101, message: 'Object not found.' }","timestamp":"2016-10-24T21:37:06.676Z"}
{"error":{"code":101,"message":"Object not found."},"level":"error","message":"Error generating response. ParseError { code: 101, message: 'Object not found.' }","timestamp":"2016-10-24T21:37:06.920Z"}
{"error":{"code":101,"message":"Object not found."},"level":"error","message":"Error generating response. ParseError { code: 101, message: 'Object not found.' }","timestamp":"2016-10-24T21:37:07.106Z"}
{"error":{"code":101,"message":"Object not found."},"level":"error","message":"Error generating response. ParseError { code: 101, message: 'Object not found.' }","timestamp":"2016-10-24T21:37:07.495Z"}
{"error":{"code":101,"message":"Object not found."},"level":"error","message":"Error generating response. ParseError { code: 101, message: 'Object not found.' }","timestamp":"2016-10-24T21:37:07.687Z"}
{"error":{"code":101,"message":"Object not found."},"level":"error","message":"Error generating response. ParseError { code: 101, message: 'Object not found.' }","timestamp":"2016-10-24T21:37:07.722Z"}
{"error":{"code":101,"message":"Object not found."},"level":"error","message":"Error generating response. ParseError { code: 101, message: 'Object not found.' }","timestamp":"2016-10-24T21:37:08.108Z"}
{"error":{"code":101,"message":"Object not found."},"level":"error","message":"Error generating response. ParseError { code: 101, message: 'Object not found.' }","timestamp":"2016-10-24T21:37:08.257Z"}
{"error":{"code":101,"message":"Object not found."},"level":"error","message":"Error generating response. ParseError { code: 101, message: 'Object not found.' }","timestamp":"2016-10-24T22:09:49.472Z"}
{"error":{"code":101,"message":"Object not found."},"level":"error","message":"Error generating response. ParseError { code: 101, message: 'Object not found.' }","timestamp":"2016-10-24T22:09:49.658Z"}
{"error":{"code":101,"message":"Object not found."},"level":"error","message":"Error generating response. ParseError { code: 101, message: 'Object not found.' }","timestamp":"2016-10-24T22:09:49.842Z"}
